packages/protocol/src/Chief.sol

```diff
@@ -22,12 +25,60 @@ import {CoreRoles} from "./access/CoreRoles.sol";
 contract Chief is CoreRoles, AccessControl, IChief {
   using Address for address;
 
-  event OpenVaultFactory(bool state);
+  /**
+   * @dev Emitted when the deployements of new vaults is alllowed/disallowed.
+   *
+   * @param allowed `true` to allow, `false` to disallow
```

Comment:
Not sure if true is variable and needs to have "`". Im fine either way btw.

Reply:
That's right, will change to "true" and "false"